### ISSUE TITLE
Card Components View change managment

### ISSUE THREAD

**Issue by sauravsaran99** [markmead/hyperui#540]
When I try to change the view of a specific card design, the view is applied to all card components. I think that if there are separate options for each card to change the view, then the changes should be applied separately.

Ref: https://www.hyperui.dev/components/marketing/product-cards
![Screenshot (46)](https://github.com/user-attachments/assets/c5575015-b1a2-474b-b609-2a5199e9df1e)

**Comment by markmead** [comment]
Yeah, I don't like that either. It was introduced as part of #534 where the pros outweighed the cons. However, I'm thinking of removing it.
